ios/browser/api/sync/prefs/brave_sync_profile_service.mm

```diff
@@ -0,0 +1,125 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import "brave/ios/browser/api/sync/prefs/brave_sync_profile_service.h"
+
+#include <unordered_map>
+
+#include "components/sync/base/model_type.h"
+#include "components/sync/base/user_selectable_type.h"
+#include "components/sync/driver/sync_service.h"
+#include "components/sync/driver/sync_user_settings.h"
+#include "ios/chrome/browser/application_context.h"
+#include "ios/chrome/browser/browser_state/chrome_browser_state.h"
+#include "ios/chrome/browser/sync/profile_sync_service_factory.h"
+#include "ios/web/public/thread/web_thread.h"
+
+#if !defined(__has_feature) || !__has_feature(objc_arc)
+#error "This file requires ARC support."
+#endif
+
+namespace brave_ios {
```

Comment:
Is there a reason we use `brave_ios` instead of nested `brave` and `ios` like in other places?

Reply:
Hmm not really, thats a mistake, fixing it now.